src/webots/Makefile

```diff
@@ -41,7 +41,7 @@ WB_PLUGINS_INCLUDE       = $(QT_CORE_INCLUDE) $(ODE_INCLUDE) -Icore -Imaths -Ino
 WB_ENGINE_INCLUDE        = $(QT_CORE_INCLUDE) $(QT_NETWORK_INCLUDE) $(ODE_INCLUDE) $(WREN_INCLUDE) -Icore -Imaths -Inodes -Inodes/utils -Iplugins -Isound -Iuser_commands -Iutil -Ivrml
 WB_CONTROL_INCLUDE       = $(QT_CORE_INCLUDE) $(QT_NETWORK_INCLUDE) -Icore -Iengine -Imaths -Inodes -Inodes/utils -Iode -Ivrml -Iutil
 # dependency of NODES on QT_GUI is necessary in WbImageTexture to retrieve the size of a texture before loading it in Wren
-WB_NODES_INCLUDE         = $(QT_CORE_INCLUDE) $(QT_GUI_INCLUDE) $(ODE_INCLUDE) $(CONTROLLER_INCLUDE) $(OIS_INCLUDE) $(FREETYPE_INCLUDE) $(WREN_INCLUDE) $(STB_INCLUDE) $(ASSIMP_INCLUDE) -Iapp -Icore -isystem external/siphash -Imaths -Inodes -Inodes/utils -Iode -Iplugins -Isound -Iutil -Ivrml
+WB_NODES_INCLUDE         = $(QT_CORE_INCLUDE) $(QT_NETWORK_INCLUDE) $(QT_GUI_INCLUDE) $(ODE_INCLUDE) $(CONTROLLER_INCLUDE) $(OIS_INCLUDE) $(FREETYPE_INCLUDE) $(WREN_INCLUDE) $(STB_INCLUDE) $(ASSIMP_INCLUDE) -Iapp -Icore -isystem external/siphash -Imaths -Inodes -Inodes/utils -Iode -Iplugins -Isound -Iutil -Ivrml
```

Comment:
I don't think that it is good to add this dependency to the Qt network from the nodes module.
I think that this code deserves a better refactoring fixing properly the relation between ``WbApplication``, ``WbNetwork`` and ``WbDownload`` classes.

Reply:
For example all the static functionalities in `WbDownload` seems that could be moved to the singleton `WbNetwork` class instead.
Doing this it would be easier to switch the signal/slot relationship between `WbNetwork` and `WbApplication`.

Reply:
Moving the `reset` and `progress` static method to `WbNetwork` would be a mess as they do not refer to `WbNetwork`, but to the `WbDownloader`. So, I don't see the point here.

Also, it would be possible to remove the dependency of `nodes/utils` on `QtNetwork`, but that would force us to create some proxy class or methods (maybe in `WbNetwork`) to access the `QtNetwork` functionality from `nodes/utils`, which is a useless complication in my point of view, given that `nodes/utils` depends on `core` which in turn depends on `QtNetwork`. So we are not introducing any new dependency there, we are just using a direct access to `QtNetwork` which is more readable that creating a proxy class or proxy methods in `core`.

If you want, we can discuss this by voice, as I am not sure to understand you point here.